From cf5e9ed7e0c3ec21021205b4a77b61e0857fccc0 Mon Sep 17 00:00:00 2001
From: Try Omarchy Release <actions@users.noreply.github.com>
Date: Thu, 1 Oct 2026 22:32:28 +0000
Subject: [PATCH] Refresh the guest package lock

clang 22.1.8-1 -> 23.1.1-1
compiler-rt 22.1.8-1 -> 23.1.1-1
harfbuzz 14.5.0-1 -> 14.5.1-1
linux 7.2.7.arch1-1 -> 7.2.8.arch1-1
linux-headers 7.2.7.arch1-1 -> 7.2.8.arch1-1
llvm-libs 22.1.8-2 -> 23.1.1-1
mesa 1:26.2.3-1 -> 1:26.2.3-2
noto-fonts 1:2026.09.01-1 -> 1:2026.10.01-1
tzdata 2026d-1 -> 2026e-1
vapoursynth 79-1 -> 80-1
vulkan-mesa-implicit-layers 1:26.2.3-1 -> 1:26.2.3-2
vulkan-swrast 1:26.2.3-1 -> 1:26.2.3-2
vulkan-virtio 1:26.2.3-1 -> 1:26.2.3-2
---
 guest/packages.lock.json | 26 +++++++++++++-------------
 1 file changed, 13 insertions(+), 13 deletions(-)

diff --git a/guest/packages.lock.json b/guest/packages.lock.json
index 140438e..8257372 100644
--- a/guest/packages.lock.json
+++ b/guest/packages.lock.json
@@ -40,8 +40,8 @@
     "cairo": "1.18.6-1",
     "cfitsio": "1:4.7.0-1",
     "chromium": "153.0.8010.52-1",
-    "clang": "22.1.8-1",
-    "compiler-rt": "22.1.8-1",
+    "clang": "23.1.1-1",
+    "compiler-rt": "23.1.1-1",
     "coreutils": "9.12-2",
     "cpio": "2.15-3",
     "cpptrace": "1.0.4-2",
@@ -151,7 +151,7 @@
     "gupnp-igd": "1.6.0-2",
     "gvfs": "1.60.3-3",
     "gzip": "1.15-1",
-    "harfbuzz": "14.5.0-1",
+    "harfbuzz": "14.5.1-1",
     "hdf5": "2.2.0-1",
     "herdr": "0.8.2-1",
     "hicolor-icon-theme": "0.18-1",
@@ -418,11 +418,11 @@
     "libzip": "1.11.4-1",
     "licenses": "20240728-1",
     "lilv": "0.28.0-1",
-    "linux": "7.2.7.arch1-1",
+    "linux": "7.2.8.arch1-1",
     "linux-api-headers": "7.2-1",
-    "linux-headers": "7.2.7.arch1-1",
+    "linux-headers": "7.2.8.arch1-1",
     "llhttp": "9.3.1-1",
-    "llvm-libs": "22.1.8-2",
+    "llvm-libs": "23.1.1-1",
     "lm_sensors": "1:3.6.2-1",
     "lmdb": "0.9.35-1",
     "localsearch": "3.11.2-1",
@@ -438,7 +438,7 @@
     "man-db": "2.13.1-2",
     "md4c": "0.5.3-1",
     "mdadm": "4.6-2",
-    "mesa": "1:26.2.3-1",
+    "mesa": "1:26.2.3-2",
     "minizip": "1:1.3.2-3",
     "mkinitcpio": "42.1-1",
     "mkinitcpio-busybox": "1.36.1-1",
@@ -457,7 +457,7 @@
     "nettle": "4.0-1",
     "networkmanager": "1.58.1-1",
     "nftables": "1:1.1.7-3",
-    "noto-fonts": "1:2026.09.01-1",
+    "noto-fonts": "1:2026.10.01-1",
     "noto-fonts-cjk": "20240730-1",
     "noto-fonts-emoji": "1:2.051-1",
     "npth": "1.8-1",
@@ -594,7 +594,7 @@
     "ttf-jetbrains-mono-nerd": "3.5.1-2",
     "ttf-liberation": "2.1.5-2",
     "ttfx": "0.3.2-1",
-    "tzdata": "2026d-1",
+    "tzdata": "2026e-1",
     "uchardet": "0.0.8-4",
     "udiskie": "2.7.0-2",
     "udisks2": "2.11.2-1",
@@ -606,14 +606,14 @@
     "uwsm": "0.27.0-1",
     "v4l-utils": "1.32.0-2",
     "v4l2loopback-dkms": "0.15.4-2",
-    "vapoursynth": "79-1",
+    "vapoursynth": "80-1",
     "vid.stab": "1.1.2-1",
     "vmaf": "3.2.1-1",
     "volume_key": "0.3.12-12",
     "vulkan-icd-loader": "1.4.357.0-1",
-    "vulkan-mesa-implicit-layers": "1:26.2.3-1",
-    "vulkan-swrast": "1:26.2.3-1",
-    "vulkan-virtio": "1:26.2.3-1",
+    "vulkan-mesa-implicit-layers": "1:26.2.3-2",
+    "vulkan-swrast": "1:26.2.3-2",
+    "vulkan-virtio": "1:26.2.3-2",
     "wayland": "1.26.0-1",
     "wayland-protocols": "1.49-1",
     "webrtc-audio-processing-1": "1.3-5",
-- 
2.55.0

